doc/SETUP.md

```diff
@@ -128,8 +127,13 @@ these, I set `GOPATH=~/gocode`.
 
 ## Other tools
 
-Some other helpful tools are listed in `config.mk`.
+If you wish to build GitHub Pages, you will also need to install the `toml` Python package:
```

Comment:
```suggestion
If you wish to build GitHub Pages from markdown, you will also need to install
the `yaml` (for kramdown-rfc) and `toml` (for mmark) Python package:
```

I think that this has to be an "and" to avoid having the script complain when you invoke it.  You could conditionally import `toml` or `yaml` in the script, but this seems easiest.

Reply:
Maybe just this instead:

```suggestion
If you wish to build GitHub Pages, you will also need to install the `toml` and `yaml` Python packages:
```